workflows/blobtoolkit.nf

```diff
@@ -117,6 +119,14 @@ workflow BLOBTOOLKIT {
 
     BUSCO_DIAMOND ( ch_genome, ch_taxon_taxa, ch_busco_db, ch_uniprot, params.blastp_outext, params.blastp_cols )
     ch_versions = ch_versions.mix ( BUSCO_DIAMOND.out.versions )
+    
+    RUN_BLASTX ( 
+        ch_genome.join(BUSCO_DIAMOND.out.first_table).map { meta, fasta, first_table -> [ meta, fasta ] },
+        ch_genome.join(BUSCO_DIAMOND.out.first_table).map { meta, fasta, first_table -> [ meta, first_table ] },
```

Comment:
Why are you doing the join and then split? Why not use the original version?

Reply:
Those transformations (join+split) would be only useful if there were multiple input genomes that would require a taxa file and identifiers to match each genome with a taxon, but as this is not implemented as far as I understood, this should work as well:

``` 
    RUN_BLASTX ( 
        ch_genome,
        BUSCO_DIAMOND.out.first_table,
        ch_blastx,
        params.blastx_outext,
        params.blastx_cols
    )
```